doc/SAI_1.6.5_ReleaseNotes.md

```diff
@@ -0,0 +1,539 @@
+8)Switch MDIO read API - Provides read access API for devices connected to MDIO from NPU SAI.<br>
+9)Switch MDIO write API - Provides write access API for devices connected to MDIO from NPU SAI.<br>
+
+```
+SAI_SWITCH_HARDWARE_ACCESS_BUS_MDIO
+SAI_SWITCH_HARDWARE_ACCESS_BUS_I2C
+SAI_SWITCH_HARDWARE_ACCESS_BUS_CPLD
+SAI_SWITCH_ATTR_FIRMWARE_LOAD_METHOD
+SAI_SWITCH_ATTR_FIRMWARE_LOAD_TYPE
+SAI_SWITCH_ATTR_TYPE
+SAI_SWITCH_ATTR_HARDWARE_ACCESS_BUS
+SAI_SWITCH_ATTR_PLATFROM_CONTEXT
+SAI_SWITCH_ATTR_REGISTER_READ
+SAI_SWITCH_ATTR_REGISTER_WRITE
+SAI_SWITCH_ATTR_FIRMWARE_DOWNLOAD_BROADCAST
+SAI_SWITCH_ATTR_FIRMWARE_LOAD_METHOD
+SAI_SWITCH_ATTR_FIRMWARE_LOAD_TYPE
+SAI_SWITCH_ATTR_FIRMWARE_DOWNLOAD_EXECUTE
+SAI_SWITCH_ATTR_FIRMWARE_BROADCAST_STOP
+SAI_SWITCH_ATTR_FIRMWARE_VERIFY_AND_INIT_SWITCH
+SAI_SWITCH_ATTR_FIRMWARE_STATUS
+SAI_SWITCH_ATTR_FIRMWARE_MAJOR_VERSION
+SAI_SWITCH_ATTR_FIRMWARE_MINOR_VERSION
+SAI_SWITCH_ATTR_PORT_CONNECTOR_LIST
+SAI_SWITCH_ATTR_PROPOGATE_PORT_STATE_FROM_LINE_TO_SYSTEM_PORT_SUPPORT
+SAI_SWITCH_ATTR_TYPE
+SAI_SWITCH_ATTR_MACSEC_OBJECT_ID
+SAI_SWITCH_ATTR_QOS_MPLS_EXP_TO_TC_MAP
+SAI_SWITCH_ATTR_QOS_MPLS_EXP_TO_COLOR_MAP
+SAI_SWITCH_ATTR_QOS_TC_AND_COLOR_TO_MPLS_EXP_MAP
+SAI_SWITCH_ATTR_SWITCH_ID
+SAI_SWITCH_ATTR_MAX_SYSTEM_CORES
+SAI_SWITCH_ATTR_SYSTEM_PORT_CONFIG_LIST
+SAI_SWITCH_ATTR_NUMBER_OF_SYSTEM_PORTS
+SAI_SWITCH_ATTR_SYSTEM_PORT_LIST
+SAI_SWITCH_ATTR_NUMBER_OF_FABRIC_PORTS
+SAI_SWITCH_ATTR_FABRIC_PORT_LIST
+
+```
+
+###saitam.h
+Definition for metadata fragment has been enabled.  When there is insufficient space in the packet to add INT metadata for this hop (e.g. MTU would be exceeded), the device may remove the metadata from the packet, send a report to the collector, and insert its metadata before forwarding the packet. SAI_TAM_REPORT_ATTR_ENTERPRISE_NUMBER has been modified for the saitam file.
+
+```
+
+SAI_TAM_INT_ATTR_METADATA_FRAGMENT_ENABLE,
+
+    /**
+     * @brief Enable checksum
+     *
+     * Enable checksum for metadata stack
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     * @validonly SAI_TAM_INT_ATTR_TYPE == SAI_TAM_INT_TYPE_IFA2
+     */
+
+
+    /**
+     * @brief Enterprise number
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 0
+     * @validonly SAI_TAM_REPORT_ATTR_TYPE == SAI_TAM_REPORT_TYPE_IPFIX
+     */
+    SAI_TAM_REPORT_ATTR_ENTERPRISE_NUMBER,
+
+    /**
+	
+```
+
+
+### saitunnel.h	
+Create and Set for Tunnel Attributes are added.
+
+```
+/**
+ * @brief Defines tunnel peer mode
+ */
+typedef enum _sai_tunnel_peer_mode_t
+{
+    /**
+     * @brief P2P Tunnel
+     */
+    SAI_TUNNEL_PEER_MODE_P2P,
+
+    /**
+     * @brief P2MP Tunnel
+     */
+    SAI_TUNNEL_PEER_MODE_P2MP,
+
+} sai_tunnel_peer_mode_t;

```

Comment:
I think the sai_tunnel.h has more diffs, than what is presented here.

Reply:
Done